docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,225 @@
+this undelegation is **not** slashed, otherwise the undelegation is [slashed](https://github.com/cosmos/cosmos-sdk/blob/bf249162e42ef084668b942e12538cb30f9e4846/x/staking/keeper/slash.go#L262) by
+`slashFactor`.
+
+The slashing of redelegations happens in a similar way, meaning that `Slash` goes through all redelegations and checks on whether
+the redelegation started before or after the `infractionHeight`.
+
+We believe that one of the ideas behind using `infractionHeight` to decide on whether to slash an undelegation or redelegation
+has to do with the fact that we want to slash only delegators whose voting power _contributed_ to the infraction.
+However, this is a rather obscure idea because a delegator `D` could start unbonding at height `H` and then a validator
+could intentionally perform an equivocation in the past (before `H`). In such a case, delegator `D` would still get slashed, even though `D`
+did not contribute any voting power per se for the equivocation. Furthermore, this principle is not respected in general
+(see "Slashing delegations" below).
+
+#### Slashing delegations
+Besides undelegations and redelegations, we need to slash simple delegations on the validator.
+This is performed by [deducting the appropriate amount of tokens](https://github.com/cosmos/cosmos-sdk/blob/5ca405ae067e7d8df98699f675e060f70a549976/x/staking/keeper/slash.go#L165)
+from the validator. Note that this deduction is computed based on the voting `power` the misbehaving validator
+had at the height of the equivocation. As a result of the tokens deduction, 
+the [tokens per share](https://docs.cosmos.network/v0.46/modules/staking/01_state.html#delegator-shares)
+reduce and hence later on, when delegators undelegate or redelegate, the delegators retrieve back less
+tokens, effectively having their tokens slashed. This approach of slashing delegations does not utilize the
+`infractionHeight` in any way and hence the following could occur:
+  1. a validator `V` performs an equivocation at a height `Hi`
+  2. a new delegator `D` delegates to `V` after height `Hi`
+  3. we receive the evidence of the equivocation by validator `V`
+  4. we slash the tokens of delegator `D`
+
+In the above scenario, delegator `D` is slashed, even though `D`'s voting power did not contribute to the infraction. 
+
+
+#### Old evidence
+In the single-chain case, we never act on old evidence (e.g., from 3 years ago). This is achieved through
+[CometBFT](https://docs.cometbft.com/v0.37/spec/consensus/evidence) that filters old evidence based on the parameters
+`MaxAgeNumBlocks` and `MaxAgeDuration` (see [here](https://github.com/cometbft/cometbft/blob/ae9826ed75ee411c0d809797ce209ee770c15c4f/evidence/pool.go#L266)).
+In Cosmos Hub, the `MaxAgeNumBlocks` is set to 1000000 (i.e., ~70 days if we assume we need ~6 sec per block) and `MaxAgeDuration`
+is set to 172800000000000 ns (i.e., 2 days). Because of this check, we can easily exclude old evidence.
```

Comment:
Are you sure about this value for `MaxAgeDuration`?

Reply:
I could not extract these values using `gaiad` (e.g., using something like `gaiad query evidence params`) so I downloaded the [genesis file](https://github.com/cosmos/mainnet/raw/master/genesis/genesis.cosmoshub-4.json.gz) found from [mintscan](https://www.mintscan.io/cosmos/parameters) and inside the file we have:
```
"evidence":
{
    "max_age_duration": "172800000000000",
    "max_age_num_blocks": "1000000",
    "max_bytes": "50000"
},
```
Looking at an arbitrary [block](https://www.mintscan.io/cosmos/blocks/16786240) we can see that BFT time is in nanosecond granularity (e.g., ` "timestamp": "2023-08-30T07:43:23.844982695Z"`) and I would expect `max_age_duration` to also be in nanoseconds because in the [code](https://github.com/cosmos/cosmos-sdk/blob/47f46643affd7ec7978329c42bac47275ac7e1cc/x/evidence/keeper/infraction.go#L45) we do:
```
ageDuration := ctx.BlockHeader().Time.Sub(infractionTime)
...
if ageDuration > cp.Evidence.MaxAgeDuration && ... 
```

Reply:
This is how you do it using `gaiad`:
```
gaiad q params subspace baseapp EvidenceParams --node https://cosmos-rpc.polkachu.com:443

key: EvidenceParams
subspace: baseapp
value: '{"max_age_num_blocks":"1000000","max_age_duration":"172800000000000","max_bytes":"50000"}'
```

EvidenceParams are under `baseapp` subspace in v45